src/util/WidgetBinding.ts

```diff
@@ -0,0 +1,20 @@
+const dataLayers = {}
+const configurations = {}
+
+export function updateBindedStores({ dataLayer, widgetId, store, actor, config }) {
```

Comment:
Could you please add some notes over here what is the purpose of adding this method?